app/src/main/res/values/strings.xml

```diff
@@ -174,6 +175,7 @@
     <string name="last_added">Last added</string>
     <string name="history">History</string>
     <string name="my_top_tracks">My top tracks</string>
+    <string name="not_played_lately">Not played lately</string>
```

Comment:
Nitpick: you're using "recently" and "lately" interchangeably. It'd be nice if you stick with one of them.

Reply:
Agree

Reply:
See efd340c